src/Drips.sol

```diff
@@ -614,28 +616,48 @@ abstract contract Drips {
             }
             configsLen = _addConfig(configs, configsLen, receiver);
         }
-        return _calcMaxEnd(balance, configs, configsLen);
+        return _calcMaxEnd(balance, configs, configsLen, maxEndTip1, maxEndTip2);
     }
 
     /// @notice Calculates the maximum end time of drips.
     /// @param balance The balance when drips have started
     /// @param configs The list of drips configurations
     /// @param configsLen The length of `configs`
     /// @return maxEnd The maximum end time of drips
-    function _calcMaxEnd(uint128 balance, uint256[] memory configs, uint256 configsLen)
-        private
-        view
-        returns (uint32 maxEnd)
-    {
+    function _calcMaxEnd(
+        uint128 balance,
+        uint256[] memory configs,
+        uint256 configsLen,
+        uint32 maxEndTip1,
+        uint32 maxEndTip2
```

Comment:
These are just binary search candidates, so it's normal that their order doesn't matter. Enforcing a relation between them would require more logic on the contract side, because we'd need to artificially `require` it and throw a new kind of error when violated. Why would we want to do that?